modules/consul-cluster/main.tf

```diff
@@ -47,6 +47,10 @@ resource "aws_autoscaling_group" "autoscaling_group" {
       var.tags,
     ]
   )
+
+  lifecycle {
+    ignore_changes = [load_balancers, target_group_arns]
```

Comment:
Could you add a comment explaining why this is here, including link(s) to the relevant docs / bugs?

Reply:
Perfect, thanks! I'll kick off tests now.